Bio/SeqFeature.py

```diff
@@ -1709,7 +1709,16 @@ def _flip(self, length):
         versions of Biopython which would have given join{[5:29](-), [37:52](-)}
         and the translation would have wrongly been "EXAMPLE*SILLY" instead.
 
+        When strand is None, the parts are reversed. In principle this does not
+        change the meaning of the location, but improves the representation of
+        locations spanning the origin in circular molecules. For more details,
+        see https://github.com/biopython/biopython/issues/4611
```

Comment:
I would leave out the issue reference, I don't think this adds anything to the documentation.

Perhaps insert "misc-features" though, as in "... improves the representation of misc-features spanning the origin in circular molecules"?

Reply:
I have done something in these lines, but I don't this is restricted to misc-features. Could be a centromere, for instance.